spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -747,7 +784,7 @@ public void testDeployWithSensitiveData() throws Exception {
 
 		final List<AuditRecord> auditRecords = auditRecordRepository.findAll();
 
-		assertEquals(2, auditRecords.size()); // TODO This looks wrong! only one Audit record should exist
+		assertEquals(2, auditRecords.size()); // TODO (Tzolov) This looks wrong! only one Audit record should exist
```

Comment:
@tzolov Could you create an issue to follow up with your reasoning as it might get missed by just having a comment here.